invenio_circulation/mappings/v2/loans/loan-v1.0.0.json

```diff
@@ -18,8 +18,7 @@
           "index": "not_analyzed"
         },
         "transaction_date": {
-          "type":   "date",
-          "format": "yyyy-MM-dd"
```

Comment:
So, dates are in this format for now:
`2018-02-01T09:30:00+02:00`
ES will automatically pick the right date format, but it could be that we will change it if it not good enough.